metadata.rb

```diff
@@ -8,8 +8,7 @@
 version '3.0.25'
 
 depends 'apache2'
-depends 'application'
-depends 'application_php'
+depends 'application', '~> 4.1.6'
```

Comment:
(a) Berkshelf is depsolving by downgrading application, so that's the need to pin. (b) I'd be okay with a `>=` pin.